akka-actor/src/main/scala/akka/Done.scala

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (C) 2009-2016 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka
+
+/**
+ * Typically used together with ```Future``` to signal completion
+ * but there is no actual value completed. More clearly signals intent
+ * than ```Unit``` and is available both from Scala and Java (which ```Unit``` is not).
```

Comment:
Oh, I only knew of the triple one. (Which is super annoying on a swedish keyboard because it wants to combine with the previous character)